envision/client_config.py

```diff
@@ -49,7 +53,4 @@ class EnvisionStateFilter:
     def default(cls):
         """Give a new default filter."""
 
-        def default_override():
-            return SingleAttributeOverride(True, None)
-
-        return cls(defaultdict(default_override), defaultdict(default_override))
```

Comment:
The core of this is that locally defined method like this becomes an out of scope reference when the standard Python pickler attempts to load it so the standard pickler just does not allow the pickle in the first place.